app/views/devise/passwords/new.html.erb

```diff
@@ -1,16 +1,20 @@
-<h2>Forgot your password?</h2>
+<div class="col-sm-8 col-sm-offset-2">
 
-<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
-  <%= devise_error_messages! %>
+  <h2>Forgot your password?</h2>
 
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true %>
-  </div>
+  <%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
```

Comment:
is `resource` how devise names it's object for this page? does it make it anonymous or something?